Sasquatch/Sasquatch.xcodeproj/xcshareddata/xcschemes/SasquatchSwift Extension.xcscheme

```diff
@@ -105,20 +105,6 @@
                ReferencedContainer = "container:../AppCenterPush/AppCenterPush.xcodeproj">
             </BuildableReference>
          </BuildActionEntry>
-         <BuildActionEntry
-            buildForTesting = "YES"
-            buildForRunning = "YES"
-            buildForProfiling = "YES"
-            buildForArchiving = "YES"
-            buildForAnalyzing = "YES">
-            <BuildableReference
-               BuildableIdentifier = "primary"
-               BlueprintIdentifier = "B21A71DA1DD2A38C0044BB1C"
-               BuildableName = "CrashLibIOS.framework"
-               BlueprintName = "CrashLibIOS"
-               ReferencedContainer = "container:../CrashLib/CrashLib.xcodeproj">
-            </BuildableReference>
-         </BuildActionEntry>
```

Comment:
Testing?

Reply:
GH multililne comments are quite cool, but not quite handy yet :(
PS feedback resolved :D